src/components/DpInput/DpInput.vue

```diff
@@ -58,9 +58,9 @@ export default {
      * you want to override the label.
      */
     ariaLabelledby: {
-      type: [String, Boolean],
+      type: [String, Boolean, null],
       required: false,
-      default: false
+      default: null
```

Comment:
if You change this to `null`, do we still need the boolean in `type` ?

Reply:
Good point! 180d240ae93b29e0d697100668095301dd45c326